pkg/auth/authenticator/bearertoken/bearertoken.go

```diff
@@ -20,16 +20,19 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/go-openapi/spec"
```

Comment:
I am for less dependencies but I think this is different. Previous one could be resolved by refactoring our code, but for this one, I need to duplicate what I need from openapi spec dependency somewhere (probably openapi/common) and import that. I can do that but not in favor of it because we need to keep two copies in sync.